### ISSUE TITLE
Maybe use podman instead of Docker

### ISSUE THREAD

**Issue by CasperWA** [EMMC-ASBL/oteapi-services#14]
Podman: https://podman.io/

Docker Desktop and Docker Hub are becoming costly options.
Podman might be the solution for replacing Docker Desktop as a Docker Engine DIY drop-in alternative. While some ease-of-use might be lost, it should be a secure and powerful alternative to the standard Docker Engine.
GitHub Packages or GitLab Container Registry or other might be the best alternative for Docker Hub.

**Comment by jesper-friis** [comment]
I think it is a good idea. But I would prefer if we can come up with a solution that allow to run podman rootless. 

However, when I try to run `PORT=8080 podman-compose up`, I get the following error:
```
ERRO[0003] error starting some container dependencies   
ERRO[0003] "rootlessport cannot expose privileged port 80, you can add 'net.ipv4.ip_unprivileged_port_start=80' to /etc/sysctl.conf (currently 1024), or choose a larger port number (>= 1024): listen tcp 0.0.0.0:80: bind: permission denied" 
```
Not sure why it uses port 80 when I explicitly asks for 8080. But it looks like a solvable problem.

**Comment by CasperWA** [comment]
> Not sure why it uses port 80 when I explicitly asks for 8080. But it looks like a solvable problem.

Indeed. Just as a reminder here - this particular thing was "solved" in #15, where the default port is now 8080 (instead of 80).
